scripts/seed_products.js

```diff
@@ -6,39 +6,37 @@ export default async ({ args }) => {
   const priceResults = []
 
   // retrieve list
-  const prices = await stripe.prices.list({
+  const products = await stripe.products.list({
     active: true,
   })
 
-  console.log(prices.length)
-
-  if (prices.length > 0) {
+  // Check length
+  if (products.data.length > 0) {
+    console.log(
+      'It looks like you already have products seeded. Archive ALL the products in your Stripe store and run the script again'
+    )
+  } else if (products.data.length === 0) {
+    //  create prices from dummy data and then update the created product
+    for (const price of dummyPrices) {
+      try {
+        const { product } = await stripe.prices.create(price.createPrice)
+        const result = await stripe.products.update(
+          product,
+          price.updateProduct
+        )
+        priceResults.push(result)
+      } catch (err) {
+        console.log(err.raw.message)
+        priceResults.push({ error: err.raw.message })
+        // If any of the price creations fail, escape the loop
+        break
+      }
+    }
```

Comment:
@paulasjes-stripe I'd like to know if theres a better way than creating prices and then immediately updating them to include a description.

Reply:
Unfortunately there's no way to do them in the same call. For legibility however I recommend just making two calls sequentially:

```js
const product = await stripe.products.create({...productOpts});
const price = await stripe.prices.create({
  product: product.id,
  ...priceOpts,
});
```

Reply:
Yeah that does look a whole bunch neater. Thanks @paulasjes-stripe